products/.gitbook_summary

```diff
@@ -39,7 +39,9 @@
 
 ## Highly Available Memgraph
 
+* [Introduction](ha_memgraph/v0.15.0/README.md)
```

Comment:
`gitbook-introduction.md` page should be linked here. The same applies to the Memgraph section.